From cf1a08458ee46bfa99b8451f80cf864802948a9c Mon Sep 17 00:00:00 2001
From: "Brian W. Hart" <hartb@us.ibm.com>
Date: Mon, 20 May 2019 13:57:26 -0500
Subject: [PATCH] Fix docstring / help string for torchscript-export

The docstring for torchscript_export() mistakenly refers to caffe2
(probably cut-n-paste from export()), and this is reflected in the
help text:

Before:

$ pytext --help
Usage: pytext [OPTIONS] COMMAND [ARGS]...
...
Commands:
  export              Convert a pytext model snapshot to a caffe2 model.
...
  torchscript-export  Convert a pytext model snapshot to a caffe2 model.

After:

  export              Convert a pytext model snapshot to a caffe2 model.
...
  torchscript-export  Convert a pytext model snapshot to torchscript.
---
 pytext/main.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pytext/main.py b/pytext/main.py
index 02ad59c..b34c7b7 100644
--- a/pytext/main.py
+++ b/pytext/main.py
@@ -367,7 +394,7 @@ def export(context, model, output_path, output_onnx_path):
 @click.option("--quantize", help="whether to quantize the model")
 @click.pass_context
 def torchscript_export(context, model, output_path, quantize=False):
-    """Convert a pytext model snapshot to a caffe2 model."""
+    """Convert a pytext model snapshot to torchscript."""
     config = context.obj.load_config()
     model = model or config.save_snapshot_path
     output_path = output_path or f"{config.save_snapshot_path}.torchscript"
-- 
1.8.3.1

